### PR TITLE
Test/issue92 automated

### PR DESCRIPTION
**Pull request by B1TC0R3**
I fixed a major issue of the bash test framework showing tests as successful because of broken regex queries.
I have fixed this issue by refactoring the tests to python, which enables me to check for patterns and changes more dynamically.
None of the tests are failing, so no additional bugs need to be fixed.

### CODE REVIEW

File: test/resources/automated_test_log [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Testing 'switch_to_horizontal_layout'...
+[A[K[32m[1mTest Success[0m - switch_to_horizontal_layout
+Testing 'move_window'...
+[A[K[32m[1mTest Success[0m - move_window
+Testing 'switch_to_tiled_layout'...
+[A[K[32m[1mTest Success[0m - switch_to_tiled_layout
+Testing 'close_window'...
+[A[K[32m[1mTest Success[0m - close_window
+Testing 'quit_oxide'...
+[A[K[32m[1mTest Success[0m - quit_oxide
```
[the remark this answers is not in the document]
Comment (FelixSchladt) [reply test/resources/automated_test_log:29]: Okay I would agree in that

File: test/resources/automated_tests.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+def oxide(command, wait_duration=1) -> str:
+    bash(oxide_msg + ' ' + command)
+    time.sleep(wait_duration)
+
+    return bash(get_state)
+
+
+def setup():
+    global oxide_msg,\
+           get_state
```
[the remark this answers is not in the document]
Comment (FelixSchladt) [reply test/resources/automated_tests.py:25]: I see where your come from, but in the light that this is just a test script, I think it can stay as is or?

**Review by FelixSchladt — APPROVED** [review]

File: test/resources/automated_tests.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (B1TC0R3) [reply test/resources/automated_tests.py:25]: Considering the size of the script, I would personally like to leave this as is. These kind of things exist for exactly the purpose I used them for, after all.

File: test/resources/automated_tests.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+    print("Running Setup...")
+
+    time.sleep(3)
```
[the remark this answers is not in the document]
Comment (B1TC0R3) [reply test/resources/automated_tests.py:29]: Can do, I suppose.

**Comment by B1TC0R3** [comment]
@gungula I have updated the test documentation and manual test documentation. Everything should be fine as is, but if you could take a quick look to make sure everything is according to doc standarts, that would be great.

I would then merge this branch, which would also be the final merge from me before the project is turned in.

**Pull request merged**